tdrs-backend/tdpservice/scheduling/test/test_file_upload.py

```diff
@@ -0,0 +1,86 @@
+"""Scheduling tests."""
+
+from datetime import datetime
+
+import pytest
+from paramiko import Transport
+from paramiko.sftp_client import SFTPClient
+from tdpservice.scheduling.sftp_task import upload
+from tdpservice.data_files.test.factories import DataFileFactory
+from tdpservice.stts.models import STT
+from django.conf import settings
+
+"""
+To mock sftp server, pytest_sftpserver (https://github.com/ulope/pytest-sftpserver) is used.
+The package provides two main fixtures for testing: sftpserver and sftpclient.
+"""
+
+@pytest.fixture
+def stt_instance(region):
+    """Return an STT."""
+    stt, _ = STT.objects.get_or_create(
+        name="first",
+        region=region,
+        code="AR",
+        stt_codee=1234,
```

Comment:
not sure if this matters for tests, but highlighting typo for awareness. 

```suggestion
        stt_code=1234,
```